pl_bolts/datamodules/sr_datamodule.py

```diff
@@ -0,0 +1,72 @@
+from pytorch_lightning import LightningDataModule
+from torch.utils.data import DataLoader, Dataset
+
+
+class SRDataModule(LightningDataModule):
```

Comment:
It seems to me that this class isn't really "SR" thing, is it...?

Reply:
Haha yes indeed, that is a good observation. What would you suggest I should do with it? 

Is it better to add the dataset preparation ([`prepare_datasets()` in `srgan/utils.py`](https://github.com/chris-clem/pytorch-lightning-bolts/blob/18132e781b42b5c0edd7daffe8cef05437b5ee96/pl_bolts/models/gans/srgan/utils.py#L30-L51)) into the DataModule class?

Reply:
@akihironitta The problem with `VisionDataModule` is that it only works with `torchvision` datasets that have a boolean `train` flag. So it would work for `MNIST`, but not for `STL10` or `CelebA` (they have a `split` flag).

I could update `VisionDataModule` first so that it works with all `torchvision` datasets and then use it here?

Reply:
Yes I can do that. At the moment, I have a bit less time then when I created these PRs but that change should be not too complicated. I try to do it next weekend.